src/features/TodoItem.js

```diff
@@ -1,5 +1,29 @@
-import React from "react";
+import React, { useState } from "react";
+import { useDispatch } from "react-redux/es/exports";
+import { changeToggle, deleteItem } from "./todoSlice";
+import "./display.css"
 
 export default function TodoItem(props) {
-  return <div>{props.todo}</div>;
+  const [toggle, setToggle] = useState(true);
```

Comment:
The word having line-through or not should depends on the value in the store, not a local variable
if you add a, b, c, strikethrough b, then remove a -> it will become c being strikethrough

Reply:
because you are using index as key, if the array is 1 less, then things are shifted by one, including your strike through
however as you todo.text is stored globally, the index didn't shift, causing a mismatch